packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -10,12 +10,16 @@ import {
   GEL_SPACING_DBL,
 } from '@bbc/gel-foundations/spacings';
 import Helmet from 'react-helmet';
-import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_B_MIN_WIDTH,
+  GEL_GROUP_CD_MIN_WIDTH,
+} from '@bbc/gel-foundations/breakpoints';
 import { getPica } from '@bbc/gel-foundations/typography';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 import { getSansRegular } from '@bbc/psammead-styles/font-styles';
 
-const MENU_ICON_SIDE_LENGTH = '2.75rem'; // 44px
+const TOP_BOTTOM_SPACING = 0.75; // 12px
```

Comment:
```suggestion
const TOP_BOTTOM_SPACING = 0.75rem; // 12px
```

Reply:
This needs to be a number so that it can be used in the calculation for the height [here](https://github.com/bbc/psammead/pull/2759/files#diff-a23047ff5909e5e673c346beeb894a3eR125).

We could do
```suggestion
const TOP_BOTTOM_SPACING = 0.75; //rem = 12px
```
to make the units clear? Let me know what you think.

Reply:
ahh okay, my bad. then I'm happy for this to just stay as it was :)